README.md

```diff
@@ -14,6 +14,7 @@ Throughout this documentation, ensure that you understand the following terminol
 - 6.5
```

Comment:
this lists should be updated now ;)

Reply:
@evgeni I think we should create a new bug for removing the support below 6.5.

Since this request is about adding support to 6.8, I did not remove those files.